lib/reporting/identity_verification_report.rb

```diff
@@ -337,23 +382,40 @@ def query
         ),
         idv_final_resolution: quote(Events::IDV_FINAL_RESOLUTION),
         fraud_review_passed: quote(Events::FRAUD_REVIEW_PASSED),
+        fraud_event_names: quote(
+          [
+            Events::FRAUD_REVIEW_PASSED,
+            Events::FRAUD_REVIEW_REJECT_AUTOMATIC,
+            Events::FRAUD_REVIEW_REJECT_MANUAL,
+          ],
+        ),
+        normalized_fraud_review_pending: "coalesce(#{[
+          # rubocop:disable Layout/LineLength
+          'properties.event_properties.fraud_review_pending',
+          'not isblank(properties.event_properties.fraud_pending_reason)',
+          'properties.event_properties.fraud_check_failed',
+          '(ispresent(properties.event_properties.tmx_status) and properties.event_properties.tmx_status not in ["threatmetrix_review", "threatmetrix_reject"])',
+          '0',
+          # rubocop:enable Layout/LineLength
+        ].join(", ")})",
       }
 
       format(<<~QUERY, params)
         fields
             name
           , properties.user_id AS user_id
           , coalesce(properties.event_properties.success, 0) AS success
-        #{issuers.present? ? '| filter properties.service_provider IN %{issuers}' : ''}
+          , coalesce(properties.service_provider, properties.event_properties.issuer) AS service_provider
```

Comment:
`issuer` is used on 'GetUspsProofingResultsJob: Enrollment status updated' (also this file itself also uses "issuer" and "service_provider" interchangeably)